cw/cw305/pyXKCP/KeccakP-1600-SnP.h

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
If I understand it correctly, most of the code under `pyXKCP` is actually from the XKCP project. It is okay to include code from other projects with compatible licenses but in OpenTitan we use the vendor tool to include such code: https://docs.opentitan.org/doc/project/detailed_contribution_guide/#vendored-code

As an example, we have used this in ot-sca to vendor in the `cw_spiflash.py` under `cw/cw305/vendor/lowrisc_opentitan` from the main OpenTitan repository. The advantage of using the vendor tool is that at any point in time, we can update the copy in the tree from upstream. the lockfile (lowrisc_opentitan.lock.hjson) is used to remember at which point in time we did the last update. Any changes, we need apply to the copy in tree are handled through patches. Please have a look at this. Further examples can be found in OpenTitan, see opentitan/hw/vendor. If you need more information, just let me know. I am happy to help.

Reply:
What I forgot to mention, you can also just specify a limited set of sources that the tool should include. This is useful in case you just need one file from a big project, e.g., `cw_spiflash.py`.

Reply:
Thank you for pointing this out. I will rework the code to be compliant with OT's vendor policy.